.travis.yml

```diff
@@ -3,12 +3,6 @@ node_js:
 - 6
 - 4
 
-matrix:
-  fast_finish: true
```

Comment:
> Not only in that case. https://docs.travis-ci.com/user/customizing-the-build/#Fast-Finishing

Again, this **only** applies if there are jobs in the matrix with allowed failures. We actually have none (the OSX one is not even currently marked as allowed-failures, we would need to have an extra line for that).